pandas/_libs/lib.pyx

```diff
@@ -53,23 +55,15 @@ from datetime cimport get_timedelta64_value, get_datetime64_value
 
 
 from tslib cimport _check_all_nulls
-import tslib
-from tslib import NaT, Timestamp, Timedelta
-import interval
+from tslib import NaT, Timestamp, Timedelta, array_to_datetime
```

Comment:
i dont' really like this but i guess its how it is currently. this is a python level dep between lib and tslib.

Reply:
Timestamp and Timedelta aren't used in lib.  A couple of modules use `lib.Timestamp` instead of `tslib.Timestamp`.  The NaT dependency could be imported directly from tslibs.nattype.  array_to_datetime... not really any good options.

Reply:
yeah, just mentioning it